calico_test/tests/st/utils/log_analyzer.py

```diff
@@ -0,0 +1,302 @@
+from datetime import datetime
+import logging
+import re
+
+from tests.st.utils.exceptions import CommandExecError
+
+_log = logging.getLogger(__name__)
+
+FELIX_LOG_FORMAT = (
```

Comment:
At present, yes.  I guess we could extend the logger to also slurp other logs, but felix is a good start, and on a par with other test frameworks.